packages/components/src/components/DetailsPanelEvent.vue

```diff
@@ -240,6 +246,7 @@ export default {
 
   methods: {
     viewEventInSequence() {
+      this.$store.commit(SET_FOCUSED_EVENT, this.object);
```

Comment:
A focused event and a selected event are different. The focused event is temporarily highlighted with teal and is not selected in the sidebar. A selected event is permanently highlighted with pink and is selected in the sidebar. 

We don't want to make the selected event the focused event, because it creates a confusing scenario where we use the focused event highlighting for the selected event. Specifically, notice how the event is highlighted in teal in the sequence diagram after clicking on the `Show in Sequence` button:

![image](https://github.com/getappmap/appmap-js/assets/45714532/b295e778-7700-4bca-ac60-7cb151d4e100)